openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/HeavyVehicleFlagEncoder.java

```diff
@@ -63,7 +65,7 @@ public HeavyVehicleFlagEncoder(PMap properties) {
         setBlockFords(properties.getBool("block_fords", true));
         setBlockByDefault(properties.getBool("block_barriers", true));
 
-        speedTwoDirections = properties.getBool("speed_two_directions", false);
+        speedTwoDirections = properties.getBool("speed_two_directions", true);
```

Comment:
Why has been the default changed from `false` to `true`? GH flag encoders seem to still fall back to `false`.

Reply:
because otherwise the graph doesn't build, we get an error " value for reverse direction would overwrite forward direction. Enable storeTwoDirections for this EncodedValue or don't use setReverse". I don't know if that property did use to get set somewhere, and that would be probably the proposed way, but on the other hand these FlagEncoder subclasses hard-code lots of profile-specific stuff anyway, so...